web/src/components/nomination/NominateForm.vue

```diff
@@ -1,36 +1,46 @@
 <template>
   <v-form class="my-3">
     <name-input
-      v-model="form.speakerName"
+      v-model="form.name"
     />
     <v-text-field
-      v-model="form.speakerEmail"
+      v-model="form.email"
       label="Email"
       outlined
     />
+    <v-text-field
+      v-model="form.photo_url"
+      label="Photo URL"
+      outlined
+    />
     <job-input
-      v-model="form.firstJob"
+      v-model="form.job"
     />
     <location-input
       v-model="form.location"
     />
     <v-textarea
-      v-model="form.speakerBio"
+      v-model="form.speaker_bio"
       label="Speaker Bio"
       hint="A brief description of the nominee"
       outlined
     />
     <topics-input v-model="form.topics" />
-
+    <urls-input
+      v-model="form.urls"
+    />
     <submitter-input
-      v-model="form.submitterInput"
+      v-model="form.submitter"
     />
     <v-checkbox
-      v-model="form.permission"
+      v-model="form.consent"
       class="mt-0"
       label="I have the speaker's permission to submit her information to the SpeakHer database."
     />
-    <v-btn color="primary">
+    <v-btn
+      color="primary"
+      @click="submit()"
```

Comment:
A better pattern for forms is to make the button have the submit property and then the form catches the submit event. See an example on my branch: 
https://github.com/WWCodeTokyo/speak-her-db/pull/47/files
in `web/src/components/nomination/NominateForm.vue`

And [this article](https://alligator.io/vuejs/vue-form-handling/)